lighthouse-core/runner.js

```diff
@@ -132,6 +132,9 @@ class Runner {
 
       /** @type {LH.RawIcu<LH.Result>} */
       const i18nLhr = {
+        // TODO(FR-COMPAT): Remove this condition when legacy runner is deprecated.
+        // If GatherContext is not collected, then we are in a non-default pass of the legacy runner.
+        gatherMode: artifacts.GatherContext ? artifacts.GatherContext.gatherMode : 'navigation',
```

Comment:
@patrickhulce would `GatherContext` make sense as a true base artifact?

Reply:
Might do this in a separate PR then

Reply:
as discussed, I'll pick this up as part of a base artifact dependency refactor